src/gravity/paris/ParisPeriodic.cu

```diff
@@ -16,9 +16,9 @@ ParisPeriodic::ParisPeriodic(const int n[3], const double lo[3], const double hi
       ddk_(2.0 * double(n[2] - 1) / (hi[2] - lo[2])),
   #elif defined PARIS_5PT
       nk_(n[2]),
-      ddi_(sqr(double(n[0] - 1) / (hi[0] - lo[0])) / 6.0),
-      ddj_(sqr(double(n[1] - 1) / (hi[1] - lo[1])) / 6.0),
-      ddk_(sqr(double(n[2] - 1) / (hi[2] - lo[2])) / 6.0),
+      ddi_(Sqr(double(n[0] - 1) / (hi[0] - lo[0])) / 6.0),
```

Comment:
`sqrt` is a library call, so this name change will break the code.

Reply:
Maybe I'm misunderstanding something, but I thought those were calls to an inline function defined in [line 7](https://github.com/cholla-hydro/cholla/blob/5bdd42b8b728e4fdc39bfc783a3d38b4e88e239a/src/gravity/paris/ParisPeriodic.cu#L7) of this file that returns the square of the argument?